edd-license-free-download.php

```diff
@@ -279,25 +279,7 @@ public static function validate_license( $license_key ) {
 	 * @return array

```

Comment:
Can you add a note to the PHPDoc about how we're returning an array for backwards compatibility purposes? So we don't forget why it's like this two years from now. :D